noir-projects/noir-protocol-circuits/crates/types/src/abis/call_request.nr

```diff
@@ -34,12 +34,6 @@ impl Empty for CallRequest {
     }
 }
 
-impl CallRequest {
-    pub fn is_empty(self) -> bool {
```

Comment:
Deleted this as the result can be different to calling `is_empty(request)`, which is confusing.